cibyl/outputs/cli/ci/system/impls/zuul/colored.py

```diff
@@ -309,10 +335,10 @@ def create_printer():
         result.add(self.palette.blue('Tenant: '), 0)
         result[-1].append(tenant.name)
 
-        if self.query >= QueryType.PROJECTS:
+        if self.query >= QueryType.PROJECTS and self.complete is not False:
```

Comment:
In that case it will not show the projects.
This may be tricky, because there's no way to know if the user set the --projects argument or not. We only have the `self.query` property.

Maybe it will be better to pass all the CI arguments to the printer and decide there what to show based on that